proto/storage/cve.proto

```diff
@@ -114,6 +121,7 @@ message CVEInfo {
   CVSSV2 cvss_v2 = 8;
   CVSSV3 cvss_v3 = 9;
   repeated Reference references = 10;
+  repeated CVSSScore cvss_info = 11;
```

Comment:
naming is hard.  a field named `info` inside a message named `info` of type named `score` feels odd.

Reply:
yes i will change it to score